deploy/fb-user-filestore-chart/templates/deployment.yaml

```diff
@@ -34,12 +34,12 @@ spec:
         # non-secret env vars
         # defined in config_map.yaml
         resources:
-          limits:
-            cpu: 300m
-            memory: 1Gi
+        #  limits:
+        #    cpu: 300m
+        #    memory: 1Gi
```

Comment:
can you delete the code rather than comment it. it'll still be in version control

Reply:
Lines removed.